src/app/data/services/local_storage_service.spec.ts

```diff
@@ -4,7 +4,7 @@
 ///<reference path="../models/content_block.ts"/>
 
 describe('LocalStorageService', function () {
-  const service = new LocalStorageService();
+  const service = new LocalStorageService(localStorage);
```

Comment:
i could imagine to introduce a context (since this does not exist use just a `describe` for it)

```js
describe('LocalStorageService', function () {
  describe('when localStorage is available', function () { 
    const service = new LocalStorageService(localStorage);

    // ... all the rest
  });
  describe('when localStorage is not available', function () { 
    const service = undefined;
        it('fetch returns and empty map if localStorage is null', function () {
      const serviceNoStorage = new LocalStorageService(null);
      const key = '1';
      serviceNoStorage.put(key, hash1);

      expect(serviceNoStorage.fetch(key)).toEqual({});
    });
});
```